geode-core/src/distributedTest/java/org/apache/geode/internal/cache/PartitionedRegionSingleHopDUnitTest.java

```diff
@@ -919,21 +897,22 @@ public void testClientMetadataForPersistentPrs() throws Exception {
 
     int locatorPort = DUnitEnv.get().getLocatorPort();
 
-    vm0.invoke(() -> createServer("disk", -1, 3, 4));
-    vm1.invoke(() -> createServer("disk", -1, 3, 4));
-    vm2.invoke(() -> createServer("disk", -1, 3, 4));
-    vm3.invoke(() -> createServer("disk", -1, 3, 4));
+    vm0.invoke((SerializableCallableIF<Integer>) this::createServer);
+    vm1.invoke((SerializableCallableIF<Integer>) this::createServer);
+    vm2.invoke((SerializableCallableIF<Integer>) this::createServer);
+    vm3.invoke((SerializableCallableIF<Integer>) this::createServer);
```

Comment:
These casts can be avoided if you use lambdas here:
```
    vm0.invoke(() -> createServer());
    vm1.invoke(() -> createServer());
    vm2.invoke(() -> createServer());
    vm3.invoke(() -> createServer());
```

Reply:
Isn't the above the intended approach of Java? It is the optimized form according to intellij

Reply:
If there was just one `createServer()` method in the class (or the zero-arg version of the method had a unique name) then we wouldn't need the cast here, but since there are multiple methods with different signatures but the same name, omitting the cast and just using a method reference results in ambiguity, since the compiler doesn't know which one you mean and throws an error.

It's true that method references are preferred in Java, but the performance optimizations are negligible in a test, and more readable code is a greater benefit here, I think. IntelliJ doesn't even highlight the lambda as replaceable with a method reference if you make this change, because I guess it knows that the compiler will complain if you do.

Reply:
I agree the casts are ugly.

Reply:
I will swap them to a lambda..

Reply:
done